Makefile

```diff
@@ -29,6 +30,13 @@ rebar:
 	chmod +x $@
 
 ui:
-	rm -rf apps/epl/priv/htdocs
-	yarn && yarn build
-	mv build apps/epl/priv/htdocs
+	@rm -rf apps/epl/priv/htdocs
+	@yarn && yarn build
+	@mv build apps/epl/priv/htdocs
+
+release:
+	@echo "Building release version of Erlang Performance Lab"
+	@make rebar
+	@make ui
+	@make
+	@./bootstrap
```

Comment:
You're right. I did this way to allow echo before everything. Is there any way to write it your way and preserve echo?